.release/ci.hcl

```diff
@@ -13,8 +13,7 @@ project "consul-api-gateway" {
 }
 
 event "merge" {
-  // "entrypoint" to use if build is not run automatically
-  // i.e. send "merge" complete signal to orchestrator to trigger build
+  // "entrypoint" to use if build is not run automatically i.e. send "merge" complete signal to orchestrator to trigger build
```

Comment:
The hclq tool runs a formatter on this file, so you'll see some changes the first time it runs. After the first time, the diff will just be whatever minor release branch is added below.